app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Show users pages</h1>
\ No newline at end of file
+<h1>User ID: <%= @user.id %> </h1>
+<h1>Name: <%= @user.name %> </h1>
+<img src=<%= @user.photo%> />
+<h3>Number of posts:<%= @user.posts_counter%> </h3>
+<hr/>
+<h4>Bio</h4>
+<p><%= @user.bio %> </p>
+<hr/>
+<% @user.recent_posts.each_with_index do |post, index| %>
+    <a href='/users/<%= @user.id %>/posts'> <h3>Post # <%= index + 1 %></h3></a>
+    <h4><%= post.text %></h4>
+    <h5>Comments:<%= post.comments_counter %></h5>
+    <h5>Likes:<%= post.likes_counter %></h5>
+    <hr/>
+<% end %> 
+<a href='/users/'><button>Go back</button></a>
```

Comment:
- [ ] This link suppose to be **See all Post**  that opens the user's post index at clicking, So please fix that.
[hint] You can use `link_to` method. See [this link](https://www.rubyguides.com/2019/05/rails-link_to-method/) for more info.

Reply:
@Omar-Muhamad Thank you for reviewing my project, I will make the necessary corrections as suggested